src/dbt/kipptaf/models/extracts/intermediate/int_extracts__student_enrollments.sql

```diff
@@ -251,4 +252,4 @@ left join
     overgrad_fafsa as ovg
     on e.salesforce_contact_id = ovg.salesforce_contact_id
     and {{ union_dataset_join_clause(left_alias="e", right_alias="ovg") }}
-where e.rn_year = 1 and e.schoolid != 999999
+where e.rn_year = 1 and e.schoolid != 999999 and e.enroll_status != -1

```

Comment:
we need to keep this enroll status for views that report out pre-registered students